multiple-choice-qs.ipynb

```diff
@@ -0,0 +1,80 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import spreadsheet cleaning loader\n",
+    "from load_survey_data import load_data, headers\n",
+    "from mcq_tables import make_table"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from datetime import datetime \n",
+    "import matplotlib\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "# Requires openpyxl for read_excel\n",
+    "from pathlib import Path\n",
+    "import re\n",
```

Comment:
Are you planning to extend this notebook later? Most of these imports are unused.

Reply:
Not at present, this is just a lazy copy and paste